packages/superset-ui-control-utils/src/InfoTooltipWithTrigger.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { kebabCase } from 'lodash';
+// @ts-ignore
+import { Tooltip, OverlayTrigger } from 'react-bootstrap';
+
+const defaultProps = {
+  className: 'text-muted',
+  icon: 'info-circle',
+  placement: 'right',
+};
+const tooltipStyle = { wordWrap: 'break-word' } as any;
```

Comment:
Try avoid `any` if possible:

```ts
const tooltipStyle: React.CSSProperties = { wordWrap: 'break-word' };
```

Since this rule is so simple, you can probably also just inline the style in `<Tooltip ... style={}>`.

Reply:
having constant here is better so react won't re-render. when passing object to `style`, `react` only compares reference.